testsuite/features/reposync/srv_sync_fake_channels.feature

```diff
@@ -93,6 +93,18 @@ Feature: Synchronize fake channels
     Then I should see a "Repository sync scheduled for Fake-Base-Channel-RH-like." text
     And I wait until the channel "fake-base-channel-rh-like" has been synced
 
+@rhlike_minion
+  Scenario: Synchronize Fake-Base-Channel-AppStream channel
+    Given I am authorized for the "Admin" section
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Fake-Base-Channel-AppStream"
+    And I follow "Repositories" in the content area
+    And I follow "Sync"
+    And I wait at most 60 seconds until I do not see "Repository sync is running." text, refreshing the page
+    And I click on "Sync Now"
+    Then I should see a "Repository sync scheduled for Fake-Base-Channel-AppStream." text
+    And I wait until the channel "fake-base-channel-appstream" has been synced
```

Comment:
LGTM, but at some point we need to review why we are waiting twice...
It seems we are waiting for the repo to be sync (60 seconds) and then for the channel to be synced (120 seconds), but actually if I'm not wrong both syncs have the same meaning, once the repo is sync, the channel is sync right?

See the implementation of `def channel_is_synced(channel)`

Anyway, not the scope of this PR, but I wanted to share my thoughts ;)

Reply:
Yep.
A synced repo should mean a synced channel, and viceversa.

IDK if we have this sequence of steps because there might be the possibility that the 1st sync does not trigger automatically and we want to be sure it is done, or just to test the (re) sync itself.
But it definitely would make sense to refactor and have the same wait time in any case - we probably do not experience issues with that just because all these dummy repos are quite small in size.